renku_notebooks/api/notebooks.py

```diff
@@ -261,26 +287,38 @@ def autosave_info(user, namespace_project):
     summary="Delete an autosave PV and or branch.",
     responses={
         204: {"description": "The autosave branch and/or PV has been deleted."},
-        404: {
-            "description": "The requested project, namespace and/or autosave cannot be found."
-        },
     },
 )
+@marshal_with(
+    ErrorResponse(),
+    code=404,
+    description="The requested project, namespace and/or autosave cannot be found.",
+)
 @authenticated
 def delete_autosave(user, namespace_project, autosave_name):
     """Delete an autosave PV and or branch."""
     if user.get_renku_project(namespace_project) is None:
         return make_response(
             jsonify(
-                {"messages": {"error": f"Cannot find project {namespace_project}"}}
+                {
+                    "error": {
+                        "message": f"Cannot find project {namespace_project}",
+                        "code": 20,
+                    }
+                }
             ),
             404,
         )
     autosave = Autosave.from_name(user, namespace_project, autosave_name)
     if not autosave.exists:
         return make_response(
             jsonify(
-                {"messages": {"error": f"The autosave {autosave_name} does not exist"}}
+                {
+                    "error": {
+                        "message": f"The autosave {autosave_name} does not exist",
```

Comment:
```suggestion
                        "message": f"The autosave branch {autosave_name} does not exist",
```
Now we're talking only about branches, right?